libraries/AP_BattMonitor/AP_BattMonitor_config.h

```diff
@@ -30,6 +30,10 @@
 #define AP_BATTERY_ESC_ENABLED AP_BATTERY_BACKEND_DEFAULT_ENABLED && HAL_WITH_ESC_TELEM
 #endif
 
+#ifndef AP_BATTERY_ESC_TELEM_OUTBOUND_ENABLED
+#define AP_BATTERY_ESC_TELEM_OUTBOUND_ENABLED AP_BATTERY_BACKEND_DEFAULT_ENABLED && HAL_WITH_ESC_TELEM && (BOARD_FLASH_SIZE > 1024)
```

Comment:
make it off by default and add to build_options.py

Reply:
done, please re-review